ci/code_checks.sh

```diff
@@ -437,6 +436,7 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
         pandas.Series.cat.rename_categories\
         pandas.Series.cat.reorder_categories\
         pandas.Series.cat.set_categories\
+        pandas.Series.dt\
```

Comment:
Can you add a comment here that we don't document the parameters of accessor methods, so we know that this is not pending as the other ignores, but something we won't be able to address.

Reply:
It's in the middle of an multiline command, how would i add a comment here?

Reply:
```suggestion
        pandas.Series.dt `# Accessors are implemented as classes, but we do not document the Parameters section` \
```

If you want, maybe this can be moved at the beginning of the list, and we can also do it for the other accessors ('str', 'cat', 'sparse`...) in this PR. Seems like for `cat` the `Parameters` section was added, you can remove it.

We can also take care of it in a separate PR, up to you.

Reply:
Thank you, it'd be better if it's taken care of in a separate PR.